Apps/WebClient/src/ClientApp/public/index.html

```diff
@@ -18,49 +18,57 @@
             name="google-site-verification"
             content="doAsS2f0kVQcQMjZcJuo2Pdj60GkhXI2iKbwfaUBImk"
         />
-
+        <script src="https://cdnjs.cloudflare.com/ajax/libs/modernizr/2.8.3/modernizr.min.js" integrity="sha512-3n19xznO0ubPpSwYCRRBgHh63DrV+bdZfHK52b1esvId4GsfwStQNPJFjeQos2h3JwCmZl0/LgLxSKMAI55hgw==" crossorigin="anonymous">
+        </script>
         <!-- Snowplow starts plowing - Standalone vE.2.14.0 -->
         <script type="text/javascript">
-            (function (p, l, o, w, i, n, g) {
-                if (!p[i]) {
-                    p.GlobalSnowplowNamespace = p.GlobalSnowplowNamespace || [];
-                    p.GlobalSnowplowNamespace.push(i);
-                    p[i] = function () {
-                        (p[i].q = p[i].q || []).push(arguments);
-                    };
-                    p[i].q = p[i].q || [];
-                    n = l.createElement(o);
-                    g = l.getElementsByTagName(o)[0];
-                    n.async = 1;
-                    n.src = w;
-                    g.parentNode.insertBefore(n, g);
+            function detectBrowser() {
+                if (!Modernizr.es6number) {
+                    const unsupportedBrowserMsg = "Health Gateway is not compatible with this browser. Please use another browser.";
+                    document.getElementById("app-root").innerHTML = unsupportedBrowserMsg;
```

Comment:
instead of adding the test to app-root I say lets remove app-root completly so that we are sure the binding to vue does not happen

Reply:
aha I like this one!

Reply:
aiee, need to find a way to remove a div in JS as the remove() is not supported in IE (worst browser ever)

Reply:
Found that removeChild works for IE yeah!